app/src/main/java/com/example/kakaowebtoon/presentation/ui/main/component/MainBottomNavBar.kt

```diff
@@ -0,0 +1,113 @@
+package com.example.kakaowebtoon.presentation.ui.main.component
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.text.PlatformTextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.kakaowebtoon.presentation.type.MainBottomNavType
+import com.example.kakaowebtoon.ui.theme.KakaoWebtoonTheme
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.toPersistentList
+
+@Composable
+fun MainBottomNavBar(
+    visible: Boolean,
+    tabs: PersistentList<MainBottomNavType>,
+    currentTab: MainBottomNavType?,
+    onTabSelected: (MainBottomNavType) -> Unit,
+    modifier: Modifier = Modifier
+) {
+    AnimatedVisibility(visible) {
+        Row(
+            modifier = modifier
+                .fillMaxWidth()
+                .padding(horizontal = 22.dp)
+                .height(43.dp),
+            horizontalArrangement = Arrangement.spacedBy(28.dp)
+        ) {
+            tabs.forEach { tab ->
+                MainBottomNavBarItem(
+                    tab = tab,
+                    selected = tab == currentTab,
+                    onTabClick = { onTabSelected(tab) }
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun RowScope.MainBottomNavBarItem(
+    tab: MainBottomNavType,
+    selected: Boolean,
+    onTabClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    val itemColor = if (selected) KakaoWebtoonTheme.colors.yellow2 else KakaoWebtoonTheme.colors.darkGrey1
```

Comment:
```suggestion
    val itemColor = if (selected)defaultKakaoWebtoonColors.yellow2 else defaultKakaoWebtoonColors.darkGrey1
```
이렇게 쓰는거랑 기존코드 중에 뭐가 더 좋을까요??

Reply:
```Kotlin
object KakaoWebtoonTheme {
    val colors: KakaoWebtoonColors
        @Composable
        @ReadOnlyComposable
        get() = LocalKakaoWebtoonColors.current

    val typography: KakaoTypography
        @Composable
        @ReadOnlyComposable
        get() = LocalKakaoWebtoonTypography.current
}
```
이렇게 싱글톤으로 생성해서 사용 하려고 만들었는데 안 쓰기엔 살짝 아쉽지 않을까요!?